ci/scripts/r_docker_configure.sh

```diff
@@ -95,5 +95,18 @@ fi
 # cmake is now a listed sys req.
 $PACKAGE_MANAGER install -y rsync cmake curl
 
+# Install libcurl (needed to compile curl, which is needed by rcmdcheck); need texlive for CRAN checks.
+# https://github.com/r-hub/containers/issues/58
+# The apt-get based images (rhub/clang17, rhub/clang18) are the only ones that seem to lack texlive right now:
+# https://github.com/r-hub/containers/issues/23
+case "$PACKAGE_MANAGER" in
+    apt-get)
+      apt-get install -y libcurl4-openssl-dev texlive-latex-base texlive-fonts-extra
```

Comment:
I've opened an issue against the rhub/containers repo to add in libcurl4-openssl. But we need this to be installed for curl to install on this machine.

texlive installing is also probably not what we want here, but it was necessary to get the build to be clean. We might consider turning off pdf manual checking on these if tex isn't available or installing tinytex in some other way.